File: datasets/IR-Plag/convertai-original-java-rest-python/case-04/plagiarized/L5/05/pengulangan.py
import math

def hit_hasil(m):
    for a in range(m, 11):
        print(f"{a}\t\t{a * 1.609}")

def main():
    m = 1
    print("Miles\t\tKilometers")
    print("-------------------------------")
    hit_hasil(m)

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/plagiarized/L5/05/pengulangan.kt
package IR

fun hitHasil(m: Int) {
    for (a in m..10) {
        println("$a\t\t${a * 1.609}")
    }
}

fun main(args: Array<String>) {
    var m = 1
    println("Miles\t\tKilometers")
    println("-------------------------------")
    hitHasil(m)
}

